src/components/drawer/index.js

```diff
@@ -341,6 +378,10 @@ class NewDrawer extends PureBaseComponent {
 export default NewDrawer;
 
 const styles = StyleSheet.create({
+  actionsContainer: {
+    flexDirection: 'row', 
+    justifyContent: 'center'
+  },
```

Comment:
Once you'll use the `animated` prop on View, you can go back to using the `row` modifier instead of this

Reply:
Done